UI/Pages/Shared/_QualityAssuredDetails.cshtml

```diff
@@ -9,6 +9,6 @@
             <li>comply with statutory safeguarding requirements</li>
             <li>offer schools a high-quality service</li>
         </ul>
-        <p class="govuk-body">You can find out more from <a href="https://www.tribalgroup.com/become-an-ntp-tuition-partner-0" class="govuk-link" data-testid="tribal-link">Tribal</a>, our quality assurance partner.</p>
+        <p class="govuk-body">You can find out more from <a href="https://www.tribalgroup.com/become-an-ntp-tuition-partner-0" rel="noreferrer noopener" target="_blank" class="govuk-link" data-testid="tribal-link">Tribal (opens in new tab)</a>, our quality assurance partner.</p>
```

Comment:
It's not part of this change, but any that open in a new tab should really state that I believe.  

I'm actually creating a new PR to review all links on the site to consider which should open in a new tab and which shouldn't, so if this shouldn't it will get resolved as part of that.